tests/typ/layout/pagebreak-parity.typ

```diff
@@ -33,3 +33,28 @@ Second
 #pagebreak(to: "odd")
 
 Third
+
+---
+// Test headers and footers are skipped on pagebreak-ed empty pages.
+#set page(
+  width: 80pt, height: 30pt, header: [header], footer: [
+    #counter(page).display("1 of 1", both: true)
+  ]
+)
+
+First
+#pagebreak(to: "odd")
+Third
+
+---
+// Test headers and footers are skipped on pagebreak-ed empty pages,
+// but with keep-marginals = true
```

Comment:
```suggestion
// but with `keep-marginals: true`.
```

Reply:
Thanks! Though I decided to combine the tests..